### PR TITLE
Added kk.html

### PR DESCRIPTION
**Pull request by krishna-ktm**
I have added this file. Please review.

### CODE REVIEW

**Review by Sanjeewaniaryal — COMMENTED** [review]
its great

**Review by ujjwal1 — COMMENTED** [review]
Reviewed.

**Review by deepakbaral — COMMENTED** [review]
This is good.

File: krish/kk.html [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+  <h2> i am krishna Okay !</h2>
```
Comment (itssabhishekk) [inline krish/kk.html:10]: ok sir

**Review by itssabhishekk — APPROVED** [review]
Great! Approved it.

**Pull request merged**